corehq/apps/locations/migrations/0002_populate_sql_locations.py

```diff
@@ -1,139 +1,14 @@
-            supply_points_index[key] = supply_point
-
-        for location in locations:
-            yield (
-                location,
-                supply_points_index.get((location['domain'], location['_id']))
-            )
 
 
 class Migration(DataMigration):
 
     def forwards(self, orm):
-        # hack: manually force sync Location design docs before
-        # we try to load from them
-        sync_docs.sync(location_models, verbosity=2)
-
-        properties_to_sync = [
-            ('location_id', '_id'),
-            'domain',
-            'name',
-            'location_type',
-            'site_code',
-            'external_id',
-            'latitude',
-            'longitude',
-            'is_archived',
-        ]
-
-        location_ids = set([r['id'] for r in Location.get_db().view(
-            'locations/by_name',
-            reduce=False,
-        ).all()])
-
-        for location, sp in iter_location_join_supply_point(location_ids):
-            try:
-                sql_location = orm.SQLLocation.objects.get(location_id=location['_id'])
-            except orm.SQLLocation.DoesNotExist:
-                # this populates bogus mptt data because otherwise
-                # null constraints will blow up but do not worry, we
-                # rebuild this at the end
-                sql_location = orm.SQLLocation.objects.create(
-                    location_id=location['_id'],
-                    lft=0,
-                    rght=0,
-                    tree_id=0,
-                    level=0
-                )
-
-            for prop in properties_to_sync:
-                if isinstance(prop, tuple):
-                    sql_prop, couch_prop = prop
-                else:
-                    sql_prop = couch_prop = prop
-
-                if couch_prop in location:
-                    setattr(sql_location, sql_prop, location[couch_prop])
-
-            if sp:
-                sql_location.supply_point_id = sp._id
-
-            # sync parent connection
-            lineage = location.get('lineage', None)
-            if lineage:
-                try:
-                    sql_location.parent = orm.SQLLocation.objects.get(location_id=lineage[0])
-                except orm.SQLLocation.DoesNotExist:
-                    # create a placeholder for the parent if it does
-                    # not yet exist, assuming that it will be properly
-                    # populated with data when its turn comes up in the
-                    # loop
-                    sql_location.parent = orm.SQLLocation.objects.create(
-                        location_id=lineage[0],
-                        lft=0,
-                        rght=0,
-                        tree_id=0,
-                        level=0
-                    )
-                    sql_location.parent.save()
-
-            sql_location.save()
-
-        # this is the important bit that rebuilds mptt tree structures
-        SQLLocation.objects.rebuild()
+        pass
```

Comment:
This file is a data migration that was needed to migrate from couch->sql back when this first happened. Really was only important on prod/staging so no longer serves enough use to continue to maintain as we work to kill `Location`. -- Tyler